lib/models/migration.js

```diff
@@ -45,6 +45,47 @@ module.exports = function(Migration, options) {
   };
 
   /**
+   * Remote Method: Run specific migration by name.
+   *
+   * @param {String} [name] Name of migration script to run.
+   * @param {String} [record] Record the migration runtime to database.
+   * @param {Function} [cb] Callback function.
+   */
+  Migration.migrateByName = function(name, record, cb) {
+    name = name || '';
```

Comment:
I think this line should be removed @rahulbile as I don't think it makes sense to default the name to an empty string.